.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,13 @@
 Thank you for contributing!
 Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
+
+If you unsure whether to check something or not leave it unchecked.
 -->
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
```

Comment:
This makes sure that the user has some specification met. Removing it will cause more CI issues.

Reply:
I've reverted it.